python/taichi/ui/scene.py

```diff
@@ -112,6 +116,19 @@ def mesh(self,
                 element indicate the RGB color of a vertex.
             two_sided (bool): whether or not the triangles should be able to be
                 seen from both sides.
+            vertex_offset: int type(ohterwise float type will be floored to int),
+                if 'indices' is provided, this means the value added to the vertex
+                index before indexing into the vertex buffer, else this means the
+                index of the first vertex to draw.
+            vertex_count: int type(ohterwise float type will be floored to int),
+                only avaliable when `indices` is not provided, which is the number
+                of vertices to draw.
+            index_offset: int type(ohterwise float type will be floored to int),
+                only avaliable when `indices` is provided, which is the base index
+                within the index buffer.
+            index_count: int type(ohterwise float type will be floored to int),
+                only avaliable when `indices` is provided, which is the the number
+                of vertices to draw.
```

Comment:
Maybe add a line or two to warn the users that the change is provisional. We have to sort out how it interacts with normals and vertex colors. Or simply add them all?